tests/farm/LiquidityMiningV1.js

```diff
@@ -1817,16 +1817,6 @@ describe("LiquidityMining", () => {
 				await expectRevert(liquidityMining.finishMigrationGracePeriod(), "Migration hasn't started yet");
 			});
 
-			it("should fail if grace period has already finished", async () => {
```

Comment:
Has this test been replaced by another one?

Reply:
this test was for when finishMigrationGracePeriod() could only be called once. If we keep that change, we could test that calling it many times doesn't revert